continue-reading.html

```diff
@@ -4,10 +4,46 @@
     <title>Continue Reading</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link href="/style3.css" rel="stylesheet" type="text/css">
   </head>
+  <style>
+    a {color: Black;}
+    a:hover {color: yellow};
+  </style>
   <body>
     <section id="header"></section>
     <section id="main"></section>
     <section id="footer"></section>

```

Comment:
If you aren't going to be using sections of the starter code, feel free to remove them to cut back on code bloat